tests/pflang-reg/pl-bug132-icmp6_or_portrange

```diff
@@ -0,0 +1 @@
+../../env pipe-lua-libpcap-match ../data/wingolog.pcap "icmp6 or portrange 14682-50101" 14951
```

Comment:
Oops, the shebangs will be added.

I thought about the env trampoline change, but that actually still breaks, because of the path to wingolog.pcap, hence writing it like this - and without the env, things under tools/ won't actually run unless we happen to be the right number of directories up from the root one, because they have hard-coded relative paths to find pf.